test/commands/remove/tag_spec.rb

```diff
@@ -41,13 +41,24 @@
       let(:tag) { "science" }
 
       it "removes tag from friend" do
-        line_changed "- Marie Curie [Atlantis] @science", "- Marie Curie [Atlantis]"
+        line_changed "- Marie Curie [Atlantis] @science",
```

Comment:
Is this just a stylistic change or am I missing something subtle here?

Reply:
Looks stylistic to me. I started breaking lines defensively to avoid `rubocop`'s warnings.

Reply:
Plus, it's easier to see what's changed if everything lines up vertically.

Reply:
Works for me! 👍